seed/views/v3/meter_readings.py

```diff
@@ -0,0 +1,54 @@
+# !/usr/bin/env python
+# encoding: utf-8
+
+from rest_framework.parsers import FormParser, JSONParser
+from rest_framework.renderers import JSONRenderer
+
+from seed.models import MeterReading, PropertyView
+from seed.serializers.meter_readings import MeterReadingSerializer
+from seed.utils.viewsets import SEEDOrgNoPatchOrOrgCreateModelViewSet
+
+
+class MeterReadingViewSet(SEEDOrgNoPatchOrOrgCreateModelViewSet):
+    """API endpoint for managing meters."""
+
+    serializer_class = MeterReadingSerializer
+    renderer_classes = (JSONRenderer,)
+    pagination_class = None
+    model = MeterReading
+    parser_classes = (JSONParser, FormParser)
+    orgfilter = 'property__organization'
+
+    def get_queryset(self):
+        # return the orgarization id from the request. This also check
+        # the permissions for the user
+        org_id = self.get_organization(self.request)
+        # get the property id - since the meter is associated with the property (not the property view)
+
+        # even though it is named 'property_pk' it is really the property view id
+        property_view_pk = self.kwargs.get('property_pk', None)
+        if not property_view_pk:
+            # Return None otherwise swagger will not be able to process the request
+            return MeterReading.objects.none()
+        property_view = PropertyView.objects.get(pk=property_view_pk)
```

Comment:
super nit: this should probably be try/catch-ed

Reply:
updated...